website/docs/help-and-support/troubleshooting-guide/deployment-errors.mdx

```diff
@@ -117,3 +120,114 @@ If you aren't receiving the invitation email, please check the value for ```APPS
 This should resolve the issue of not receiving the invitation email. Additionally, it may be helpful to verify that the email server being used is working correctly and there are no issues with the network or other components preventing the emails from being sent.
 
 However, if you encounter any issues, you can contact the support team on [Discord](https://discord.com/invite/rBTTVJp) or ask questions on the [community forum](https://community.appsmith.com).
+
+## Server shuts down with schema mismatch error
+
+The instance keeps restarting due to a server shutdown displaying a schema mismatch error.
+
+#### Error message
+
+```bash
+Looks like you skipped some required update(s), please go back to the mandatory upgrade path v1.9, or refer to ''https://docs.appsmith.com/'' for more info
```

Comment:
@harshilp24 - Why are we having a link to `docs.appsmith.com` here?

Reply:
Also please use <Message> component to highlight the error message.

Reply:
@jnikhila it is part of the whole error message? Should I remove docs.appsmith part?

Reply:
yes, please.